### ISSUE TITLE
https 'mixed content' warning

### ISSUE THREAD

**Issue by PaulBoon** [DANS-KNAW/dccd-webui#15]
When using https on the dendro site (when in place) we get an 'mixed content' warning in the browser. 
The links on the site are sometimes with plain http, which causes that problem. 
The fix is to change http url's to https versions when possible.

**Comment by PaulBoon** [comment]
The following url's need to be fixed:
- On the Home page we have http://www.mozilla.com/, change to https://www.mozilla.com/
- The map has a link to http://www.openstreetmap.org/copyright, change to https://www.openstreetmap.org/copyright
- top menu FAQs link is http://vkc.library.uu.nl/vkc/dendrochronology/research/ProjectsWiki/DCCD%20Repository%20FAQs.aspx, change to 
  https://vkc.uu.nl/vkc/dendrochronology/research/ProjectsWiki/DCCD%20Repository%20FAQs.aspx

Note that for VKC links we also fix issue #10 
- 'Products and links' page contains several VKC links, https://vkc.uu.nl/vkc/dendrochronology/research/ProjectsWiki/DCCD%20Products%20TRiDaBASE, https://vkc.uu.nl/vkc/dendrochronology/Pages/Default.aspx
- links to www.tridas.org can not be fixed right now

**Comment by PaulBoon** [comment]
Just discovered that the external links are not the real problem, although it's better to use https if possible anyway. 
The main problem are the map tile images!

The openstreetmap tiles now are referred as, for example, `http://a.tile.openstreetmap.org/3/4/2.png`, 
no https. 

So instead of using the default method in the Javascript code for OpenLayers: 
`OpenLayers.Layer.OSM`
The https version needs to be explicitly specified: 

```
OpenLayers.Layer.OSM("OpenStreetMap",
  ["https://a.tile.openstreetmap.org/${z}/${x}/${y}.png",
   "https://b.tile.openstreetmap.org/${z}/${x}/${y}.png",
   "https://c.tile.openstreetmap.org/${z}/${x}/${y}.png"]);
```
